src/components/MAPI/apps/AppInstallModal.tsx

```diff
@@ -552,7 +572,15 @@ const AppInstallModal: React.FC<
           break;
 
         default:
-          errorMessage = `Something went wrong while trying to install your app. Please try again later or contact support: support@giantswarm.io`;
+          errorMessage = (
+            <>
+              Something went wrong while trying to install your app. Please try
+              again later or contact support: support@giantswarm.io
+              <Text size='small' as='div' margin={{ top: 'medium' }}>
+                {metav1.getStatusMessage((error as GenericResponseError)?.data)}
```

Comment:
In other places, we extract the error message from a k8s API response using the `extractErrorMessage` function from `src/components/MAPI/utils.ts`. Would it be possible to also do that here?

Reply:
oh, good point, thank you!